releases/v1.4/schedule.md

```diff
@@ -0,0 +1,19 @@
+# KubeVirt 1.4 Release Schedule
+
+| **When**      | **Milestone**      | **KubeVirt**                   | **KubeVirt CI**                  | **Kubernetes**    |
+|---------------|--------------------|--------------------------------|----------------------------------|-------------------|
+| **July**      |                    |                                |                                  |                   |
+| 2024-07-16    |                    |                                |                                  | Tag 1.31.0-beta.0 |
+| 2024-07-30    |                    |                                | Start the 1.31 provider          |                   |
+| **August**    |                    |                                | Add on-demand pre-submit         |                   |
+| 2024-08-13    | Kubernetes Release |                                |                                  | 1.31.0 release    |
+| 2024-08-27    | Alpha 0 (1.4)      | Tag v1.4.0-alpha.0             |                                  |                   |
+| **September** |                    |                                |                                  |                   |
+| 2024-09-10    |                    |                                | CI lanes for provider are voting |                   |
+| 2024-09-17    | Beta 0             | Tag v1.4.0-beta.0              | Mandatory pre-submit             |                   |
+| **October**   |                    |                                |                                  |                   |
+| 2024-10-22    | Feature Freeze     | Branch release-1.4             |                                  |                   |
+| 2024-10-29    | RC 0               | Tag v1.4.0-rc.0 on release-1.4 |                                  |                   |
+| **November**  |                    |                                |                                  |                   |
+| 2024-11-04    | RC 1               | Tag v1.4.0-rc.1 on release-1.4 |                                  |                   |
+| 2024-11-11    | GA                 | Tag v1.4.0 on release-1.4      |                                  |                   |
```

Comment:
```suggestion
| 2024-11-05    | RC 1               | Tag v1.4.0-rc.1 on release-1.4 |                                  |                   |
| 2024-11-12    | GA                 | Tag v1.4.0 on release-1.4      |                                  |                   |
```
@fossedihelm I appreciate your thinking here but maintaining our Tuesday milestones and releasing on Day 1 of KubeCon is perfectly fine, and probably better from the cncf PR pov. Especially if the release is handled in the EMEA timezone and before everything kicks off. Thank you!

Reply:
Fine, thank you for the feedback